boards/st/nucleo_g431kb/nucleo_g431kb.dts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2024 Klaus Nagel, <nagelkl01@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/g4/stm32g431Xb.dtsi>
+#include <st/g4/stm32g431k(6-8-b)tx-pinctrl.dtsi>
+
+/ {
+	model = "STMicroelectronics STM32G431KB-NUCLEO board";
+	compatible = "st,stm32g431kb-nucleo";
+
+	chosen {
+		zephyr,console = &stlink_serial;
+		zephyr,shell-uart = &stlink_serial;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+		green_led: led_0 {
+			gpios = <&gpiob 8 GPIO_ACTIVE_HIGH>;
+			label = "User LD2";
+		};
+
+	};
+
+	aliases {
+		led0 = &green_led;
+		watchdog0 = &iwdg;
+	};
+};
+
+&clk_lsi {
+	status = "okay";
+};
+
+&clk_hsi {
+	status = "okay";
+};
+
+/* we adjust the pll for a SYSTEM Clock of 170MHz
+ */
```

Comment:
remove talking as a person, put on single like, use capital letter for first letter